service/config/data-model.yml

```diff
@@ -0,0 +1,3 @@
+default:
+  database:
+    users : ['Ale', 'Angie', 'Anto', 'Chechu', 'Cielo', 'Juampi', 'Kako', 'Kathy', 'Nico', 'Tuli', 'Omar']
```

Comment:
There is no official list of the names to add. I need documentation to perform my best.